Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -149,6 +149,12 @@ export async function run(artifactToolPath: string): Promise<void> {
             retries++; 
         }
 
+        if(publishStatus === packageAlreadyExistsError){
+            throw new Error(tl.loc("Error_PublishingRetry",
```

Comment:
I dont think we need a new string for the error, the debug log will show package already exist error, we can throw it the same way as the[ old implementation ](https://github.com/microsoft/azure-pipelines-tasks/blob/92ae148a02b1a174cc212222f9d92b07dc536ea6/Tasks/UniversalPackagesV0/universalpublish.ts#L204)

Reply:
If I change the Error to Error_UnexpectedErrorArtifactTool I will need to give the error message that is not passed to the outside scope from publishPackageUsingArtifactTool. Adding this new error solves this issue and also gives more insight.

Reply:
The old string already contains the exit code from artifacttool and the error message of `package already exist`

These are the[ exit code](https://dev.azure.com/mseng/AzureDevOps/_git/ArtifactTool?path=%2Fsrc%2FArtifactTool%2FErrorCode.cs) that get returned from artifact tool. Throwing using the current string we have should capture both the exit code as well as the type of error. Look [here](https://github.com/microsoft/azure-pipelines-tasks/blob/a009bb0cd3a2b17b83770826bbf37c0d187a90bd/Tasks/UniversalPackagesV0/universalpublish.ts#L210).

Reply:
removed extra error template